bundles/binding/org.openhab.binding.onewire/src/main/java/org/openhab/binding/onewire/internal/InterfaceAbstractOneWireBindingConfig.java

```diff
@@ -0,0 +1,7 @@
+package org.openhab.binding.onewire.internal;
+
+import org.openhab.core.binding.BindingConfig;
+
+public interface InterfaceAbstractOneWireBindingConfig extends BindingConfig {
+
+}
```

Comment:
I have next to the item bindings for OneWireDevices also an Item Binding for the control of the OneWire-bindings implemented (eg, the selective reset of the cache).
The two item bindings are fundamentally different, but I wanted a common interface for these two item bindings so that I can work with it in general methods.

e.g. in OneWireBindingProvider: public InterfaceAbstractOneWireBindingConfig getBindingConfig (String pvItemName);